src/ml/neural_net/style_transfer/mps_style_transfer_utils.m

```diff
@@ -487,12 +488,45 @@ + (void) populateMean:(NSMutableData *)data {
   }
 }
 
+// Note that populateMultiplication will populate data in HWC format
 + (void) populateMultiplication:(NSMutableData *)data {
   NSUInteger dataSize = (data.length)/sizeof(float);
   NSAssert((dataSize) % 3 == 0, @"Data must follow a 3 channel format");
   for(NSUInteger x = 0; x < dataSize; x++)
     ((float *)data.mutableBytes)[x] = 255.0;
 }
 
++ (void)populateMeanCWH:(NSMutableData *)data height:(NSUInteger)height width:(NSUInteger)width {
+  NSUInteger dataSize = (data.length) / sizeof(float);
+  NSAssert((dataSize) % 3 == 0, @"Data must follow a 3 channel format");
+  float meanWeights[3] = {0.485, 0.456, 0.406};
```

Comment:
I guess, it would probably be best if this was const.

Reply:
Maybe I don't understand what this method is suppose to do. Are these values just stubs for now?